scripts/std-lib-regression.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Test for platform
+platform=`uname -sp`
+if [[ $platform != "Linux x86_64" ]]; then
+  echo "Codegen script only works on Linux x86 platform"
+  exit 0
+fi
+
+# Get RMC root
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+RMC_DIR=$SCRIPT_DIR/..
+
+# Log output
+STD_LIB_LOG="/tmp/SizeAndAlignOfDstTest/log.txt"
+
+# Use a unit test that requires mutex and cell
+cd /tmp
+if [ -d SizeAndAlignOfDstTest ]; then rm -rf SizeAndAlignOfDstTest; fi
+cargo new SizeAndAlignOfDstTest
+cd SizeAndAlignOfDstTest
+cp $RMC_DIR/src/test/cbmc/SizeAndAlignOfDst/main_fail.rs src/main.rs 
+rustup component add rust-src --toolchain nightly
```

Comment:
CI shows the following:
```
      Created binary (application) `SizeAndAlignOfDstTest` package
error: toolchain 'nightly-x86_64-unknown-linux-gnu' is not installed
Starting RMC codegen for the Rust standard library
```
Yet the script seems to be passing. Maybe you can remove this line?

Reply:
piped this part to dev null and printing on success, thanks!